src/main/java/ddingdong/ddingdongBE/domain/form/entity/FormResponse.java

```diff
@@ -0,0 +1,49 @@
+package ddingdong.ddingdongBE.domain.form.entity;
+
+import ddingdong.ddingdongBE.common.BaseEntity;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class FormResponse extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private LocalDateTime submittedAt;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String studentNumber;
+
+    @Column(nullable = false)
+    private String department;
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private FormResponseStatus status;
```

Comment:
p3) 클래스명과 다른 컬럼명을 가지게 한다면, column명을 지정해주는 것이 헷갈리지 않을 것 같아요
```suggestion
    @Column(nullable = false, name = "status")
    private FormResponseStatus status;
```

Reply:
반영 완료하였습니다!